### PR TITLE
implementing dmn guide

### PR DESCRIPTION
**Pull request by christinaausley**
First draft of DMN guide. Still need to...

1. Provide testing information following deployment (currently discussing best means for this with Niall. (Completed by EOD April 26)
2. Add screenshot of blank DMN table. (Completed by EOD April 25)

### CODE REVIEW

**Comment by christinaausley** [comment]
@akeller I've updated screenshots and added a note teasing business rule tasks, but note that Niall and I have been working together to understand post-deployment within DMN so I can more clearly outline this process. Currently, we don't have this info within C8 docs (as in, ok, I've deployed my DMN model, now how do I practically implement/use it?)

Niall and I have a meeting set for Friday morning to review, so I can add more details to the end of this doc and we can get this merged before you're away for FTO.

**Comment by akeller** [comment]
@christinaausley I have no further feedback to provide on this PR at this point, so if you aren't able to merge while I'm around know that you absolutely still can. I'm also anticipating an issue from engineering on revisions to DMN content in the C8 docs because it sounds like sounds like a rewrite is necessary (almost all docs were pulled straight from C7).

**Comment by christinaausley** [comment]
@akeller This is revived! Feel free to take one more glance over to officially deem this "Complete".

File: docs/guides/create-decision-tables-using-dmn.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Set up
+
+### Create a Camunda Platform 8 account
+
+We'll begin building our DMN diagrams with [Modeler](../components/modeler/about.md). To get started, ensure you've [created a Camunda Platform 8 account](./create-account.md).
+
+### Create a cluster
+
+import CreateCluster from './assets/react-components/create-cluster.md'
```
Comment (akeller) [inline docs/guides/create-decision-tables-using-dmn.md:25]: Strange, this didn't render for me when I ran it locally. When this is merged, let's make sure we can see it.

**Review by akeller — APPROVED** [review]
I'm good with where this ends. It gives us a great starting point for the next DMN guide or more complex BPMN modeling guides. 

I left a few comments, but none of them are "blocking" in my mind.

**Comment by christinaausley** [comment]
@akeller Sorry to re-request review again, but figured I could try to address your more pressing comment with an admonition. If I didn't answer your question, I'll go ahead and get this merged and open up a new PR.

**Review by akeller — APPROVED** [review]
Mighty **morphing** power rangers 💪

**Pull request merged**